src/main/java/com/contrastsecurity/http/UrlBuilder.java

```diff
@@ -102,9 +102,41 @@ public String getTracesByApplicationUrl(String organizationId, String appId, Tra
         return String.format("/ng/%s/traces/%s/filter/%s", organizationId, appId, formString);
     }
 
+    public String getRecommendationByTraceId(String organizationId, String traceId){
+        return String.format("/ng/%s/traces/%s/recommendation", organizationId, traceId);
+    }
+
+    public String getStoryByTraceId(String organizationId, String traceId){
+        return String.format("/ng/%s/traces/%s/story", organizationId, traceId);
+    }
+
+    public String getHttpRequestByTraceId(String organizationId, String traceId){
+        return String.format("/ng/%s/traces/%s/httprequest", organizationId, traceId);
+    }
+
+    public String getEventSummary(String organizationId, String traceId){
+        return String.format("/ng/%s/traces/%s/events/summary", organizationId, traceId);
+    }
+
+    public String getEventDetails(String organizationId, String traceId, String eventId){
+        return String.format("/ng/%s/traces/%s/events/%s/details", organizationId, traceId, eventId);
+    }
+
     public String getNotesByApplicationUrl(String organizationId, String appId, String traceId, TraceFilterForm form) throws UnsupportedEncodingException {
         String formString = form == null ? "" : form.toQuery();
-        return String.format("/ng/%s/applications/%s/traces/%s/notes?expand=skip_links", organizationId, appId, traceId, formString);
```

Comment:
Why did you remove the expand=skip_links?

Reply:
is that no longer needed?

Reply:
Yeah, it wasn't necessary for this endpoint

Reply:
This endpoint, as far as I know, doesn't return links, so it shouldn't need the expand param